src/Home.tsx

```diff
@@ -307,6 +306,7 @@ const Home = (props: HomeProps) => {
   ) => {
     try {
       setIsUserMinting(true);
+      console.log(`document.getElementById("#identity"): ${document.getElementById("#identity")}`);
```

Comment:
should be removed

Reply:
good catch, should I also remove the click below?